elk/run.py

```diff
@@ -178,15 +181,39 @@ def apply_to_layers(
         with ctx.Pool(num_devices) as pool:
             mapper = pool.imap_unordered if num_devices > 1 else map
             df_buffers = defaultdict(list)
-
+            layer_outputs = []
             try:
-                for df_dict in tqdm(mapper(func, layers), total=len(layers)):
-                    for k, v in df_dict.items():
+                for df_dict, layer_output in tqdm(
```

Comment:
This doesn't write all the appropriate lines for 

❯ elk elicit gpt2 imdb amazon_polarity --max_examples 10 300 --debug --num_gpus 1

There should be evaluation results for both imdb and amazon_polarity in the layer_ensembling_results.csv